bundles/org.openhab.binding.serial/src/main/java/org/openhab/binding/serial/internal/handler/CommonBridgeConfiguration.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
```

Comment:
```suggestion
/*
 * Copyright (c) 2010-2025 Contributors to the openHAB project
```

for all added files.